### ISSUE TITLE
TPOTRegressor error

### ISSUE THREAD

**Issue by vumichien** [EpistasisLab/tpot#1066]
This is code based on tpot.py. I think something wrong here.
```
class TPOTRegressor(TPOTBase):
    """TPOT estimator for regression problems."""

    scoring_function = 'neg_mean_squared_error'  # Regression scoring
    default_config_dict = regressor_config_dict  # Regression dictionary
    classification = False
    regression = True

    def _init_pretest(self, features,target):
        """Set the sample of data used to verify pipelines work with the passed data set

        """
        self.pretest_X, _, self.pretest_y, _ = train_test_split(X, y, random_state=self.random_state,
                                                                test_size=None, train_size=min(50,int(0.9*features.shape[0])))
```
When I run TPOTRegressor, it throws the error **NameError: name 'X' is not defined**. I think X and y in 
```
 train_test_split(X, y, random_state=self.random_state, test_size=None, train_size=min(50,int(0.9*features.shape[0])))
```
should be **features** and **target.** Please help me check. Thank you

**Comment by ivan-sparq** [comment]
Replicated: problem exists on version `0.11.2` but not `0.11.1`

**Comment by weixuanfu** [comment]
Thank you for catching and reporting this bug. I will push a patch soon.

**Comment by weixuanfu** [comment]
I released TPOT v0.11.3a0 with the fix. Please upgrade TPOT or downgrade TPOT to v0.11.1 . I closed the issue for now. Please feel free to reopen this issue if the problem still exists.
